linera-service/src/cli_wrappers/local_net.rs

```diff
@@ -366,6 +390,13 @@ impl LocalNet {
 
     async fn run_server(&mut self, i: usize, j: usize) -> Result<Child> {
```

Comment:
That's validator `i`, shard `j`, right? We should document that, or rename the parameters.

Reply:
We can do that.
Do you want this in this PR?

Reply:
No, doesn't have to be in this PR. :+1: